examples.js

```diff
@@ -0,0 +1,107 @@
+function examples() {
+
+  setTimeout(blastOff, 86400000);
```

Comment:
¿Qué es 86400000? Podrías definirlo de forma más clara en una variable
```suggestion
  const millisecondsInDay = 24 * 60 * 60 * 1000;
  setTimeout(blastOff, millisecondsInDay);
```